packages/nodes-base/nodes/CustomerIo/CustomerIoTrigger.node.ts

```diff
@@ -0,0 +1,310 @@
+import {
+	IHookFunctions,
+	IWebhookFunctions,
+} from 'n8n-core';
+import {
+	INodeTypeDescription,
+	INodeType,
+	IWebhookResponseData,
+} from 'n8n-workflow';
+import {
+	apiRequest,
+} from './GenericFunctions';
+
+export class CustomerIoTrigger implements INodeType {
+	description: INodeTypeDescription = {
+		displayName: 'Customer.io Trigger',
+		name: 'customerIo',
+		group: ['trigger'],
+		icon: 'file:customer.Io.png',
+		version: 1,
+		description: 'Starts the workflow on a Customer.io update.',
+		defaults: {
+			name: 'CustomerDotIO Trigger',
+			color: '#00FF00',
+		},
+		inputs: [],
+		outputs: ['main'],
+		credentials: [
+			{
+				name: 'customerIoApi',
+				required: true,
+			}
+		],
+		webhooks: [
+			{
+				name: 'default',
+				httpMethod: 'POST',
+				responseMode: 'onReceived',
+				path: 'webhook',
+			},
+		],
+		properties: [
```

Comment:
Other webhooks separate their events into source and event, and I think it might make sense to do that here. Check out active campaign's trigger node to see this